test/mpi/test_mpi.cc

```diff
@@ -39,25 +39,9 @@ static ucc_status_t oob_allgather_free(void *req)
 }
 
 UccTestMpi::UccTestMpi(int argc, char *argv[], ucc_thread_mode_t _tm, int is_local) {
-    int required = (_tm == UCC_THREAD_SINGLE) ? MPI_THREAD_SINGLE
-        : MPI_THREAD_MULTIPLE;
-    int size, provided;
```

Comment:
@karasevb have tries to compile after rebase? Looks like you removed "size" which is used below in ctx params?

Reply:
fixed